documentation/memory/step3/using_gc_traces.md

```diff
@@ -1 +1,23 @@
 //TODO
+
```

Comment:
Didn't remove the TODO as this still feels like a stub. 
What else would go into this file?

Reply:
can we add a paragraph that explains the trace gc output itself, guiding how to interpret those?

Reply:
It's quite a rabbithole. I'd like to finish this PR and dive into that in the next. There's a lot to know there and I feel like I need to learn some more details before I'm comfortable drawing the line and deciding what goes here as enough of a summary.